pypeit/par/pypeitpar.py

```diff
@@ -703,6 +703,17 @@ def __init__(self, method=None, pixelflat_file=None, spec_samp_fine=None,
                                           'relative weights is given by ``nspec/slit_illum_smooth_npix``, ' \
                                           'where nspec is the number of spectral pixels.'
 
+        defaults['flatfield_structure'] = False
```

Comment:
Maybe a better name for this parameter that conveys that you are doing a 2D fit or something. The 2d bispline is not a real 2d fit. Just flatfield_structure does not seem to get across what is done differently here.

Reply:
OK - I've updated this based on our Slack discussion.